project.clj

```diff
@@ -15,7 +15,7 @@
        ns
        "\u001B[35m]\u001B[33m λ\u001B[m=> "))
 
-(defproject gov.nasa.earthdata/cmr-service-bridge "1.6.0-SNAPSHOT"
+(defproject gov.nasa.earthdata/cmr-service-bridge "1.6.1-SNAPSHOT"
```

Comment:
It's okay that you bumped this (and probably best that you did), but just so you're aware of the underlying usage, it isn't strictly necessary ... nothing uses cmr-service-bridge as a dep, and Bamboo clones it (doesn't pull it from Clojars).

Again, +1 for the micro-bump ... just FYI.

Reply:
bump all the things